reportportal/values.yaml

```diff
@@ -200,6 +210,7 @@ postgresql:
     user: rpuser
     dbName: reportportal
     password:
+    connections:
```

Comment:
I can't find where you use this.

Reply:
I found this value here: [tempates/api-deployment.yaml](https://github.com/reportportal/kubernetes/blob/develop/reportportal/templates/api-deployment.yaml#L76)